### PR TITLE
feat(chat): query to list channels

### PR DESCRIPTION
**Pull request by ltacker**
Query to list all channels
This query is also concerned by: #23

### CODE REVIEW

File: x/chat/types/keys.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ const (
 	ChannelCountKey = "channel_count"
 
 	// ChannelKey is the key to store the channels
-	ChannelKey = "channel"
+	ChannelKey = "channel_store"
```
Comment (fadeev) [inline x/chat/types/keys.go:25]: Is there a reason, why we can't use `channel-` as a prefix?

**Review by fadeev — APPROVED** [review]
🚀

File: x/chat/types/keys.go (hunk 1 above)
Comment (ltacker) [reply x/chat/types/keys.go:25]: That would work, you just need to have a name that is not a prefix of another key name

**Review by ilgooz — APPROVED** [review]
:wine_glass:

**Pull request merged**